### PR TITLE
aspell: split library into a subpkg

### PR DESCRIPTION
**Pull request by CameronNemo**

### CODE REVIEW

**Comment by ericonr** [comment]
Isn't it bad if people have this installed and lose the tools?

**Comment by CameronNemo** [comment]
Maybe, but how often do people manually install aspell and use these tools? I think it is likely pulled in by other packages in most cases, or used for the main program (aspell) and no others. This is one of the few packages that pulls in Perl on my inst

**Comment by sgn** [comment]
I take cnemo's side.

**Comment by Duncaen** [comment]
Wouldn't it make more sense to split it into a libaspell package?
So users who manually installed it can keep it, others will have both until everything is rebuild against the new dependency name and then the aspell package gets orphaned.

**Comment by CameronNemo** [comment]
Changed this to split libaspell out.

**Review by Duncaen — APPROVED** [review]

File: common/shlibs [hunk 1]
```diff
@@ -554,8 +554,8 @@ libicudata.so.67 icu-libs-67.1_1
 libicutu.so.67 icu-libs-67.1_1
 libicuuc.so.67 icu-libs-67.1_1
 libicutest.so.67 icu-libs-67.1_1
-libaspell.so.15 aspell-0.60_1
-libpspell.so.15 aspell-0.60_1
+libaspell.so.15 libaspell-0.60.8_2
+libpspell.so.15 libaspell-0.60.8_2
```
Comment (Duncaen) [inline common/shlibs:558]: I guess you could remove the duplicated line, I don't think there is a reason for that.

Comment (CameronNemo) [reply common/shlibs:558]: libaspell vs libpspell

Comment (Duncaen) [reply common/shlibs:558]: oh sorry, looks good.

**Pull request merged**